code/modules/robotics/bot/secbot.dm

```diff
@@ -49,21 +65,20 @@
 
 	var/auto_patrol = 0		// set to make bot automatically patrol
 
-	var/beacon_freq = 1445		// navigation beacon frequency
-	var/control_freq = 1447		// bot control frequency
-
+	var/beacon_freq = 1445	// navigation beacon frequency
+	var/control_freq = 1447	// bot control frequency
 
 	var/turf/patrol_target	// this is turf to navigate to (location of beacon)
 	var/new_destination		// pending new destination (waiting for beacon response)
 	var/destination			// destination description tag
 	var/next_destination	// the next destination in the patrol route
 	var/list/path = null	// list of path turfs
 
-	var/moving = 0 //Are we currently ON THE MOVE?
+	var/moving = 0 			//Are we currently ON THE MOVE?
 	var/current_movepath = 0
 	var/datum/secbot_mover/mover = null
-	var/arrest_move_delay = 2.5
-	var/emag_stages = 2 //number of times we can emag this thing
+	var/arrest_move_delay = 1
```

Comment:
*concern*

Reply:
I forgot I set that. Also couldn't tell if it did anything. I'll fix it.